--- epoch2d/Makefile_old	2021-07-15 23:41:44.000000000 +0800
+++ epoch2d/Makefile	2022-02-22 12:00:58.415929349 +0800
@@ -204,7 +204,7 @@
 #DEFINES += $(D)PARTICLE_SHAPE_TOPHAT
 
 # Use fifth order particle weighting (default is third order).
-#DEFINES += $(D)PARTICLE_SHAPE_BSPLINE3
+DEFINES += $(D)PARTICLE_SHAPE_BSPLINE3
 
 # Include a unique global particle ID. The first flag defines the ID using
 # an 8-byte integer, the second uses 4-bytes.
@@ -212,7 +212,7 @@
 #DEFINES += $(D)PARTICLE_ID4
 
 # Include QED routines
-#DEFINES += $(D)PHOTONS
+DEFINES += $(D)PHOTONS
 
 # Use the Trident process for pair production
 #DEFINES += $(D)TRIDENT_PHOTONS
